File: chap3/Terminaison.py
# Ces programmes sont sous licence CeCILL-B V1.
s = 2
p = False
while not p:
   # Nous rappelons que range(1,s) signifie [1; s[
   for i in range(1,s):
      j = s - i
      if i * i == 25 * j * j:
         print(i,j)
         p = True
   s = s + 1

File: chap3/Terminaison.java
// Ces programmes sont sous licence CeCILL-B V1.
class Terminaison {

  public static void main (String [] args) {
    int s,i,j;
    boolean p;
    s = 2;
    p = false;
    while (!p) {
      for (i = 1; i <= s - 1; i = i + 1) {
        j = s - i;
        if (i * i == 25 * j * j) {
          System.out.print(i);
          System.out.print("  ");
          System.out.println(j);
          p = true;}}
      s = s + 1;}}}

